lib/roles_conversion.rb

```diff
@@ -1,65 +1,67 @@
 class RolesConversion
```

Comment:
I find it awkward to do private class methods in Ruby - I've never liked the `private_class_method` convention and here we were just keeping all of the class methods public when we were really needed one (and that method called the rest).

In my mind, methods that are only used for internal logic in a class should be private (keeps the public interface of the class small), and since it's easier to do private instance methods, I usually do that.